src/helpers/Templates.js

```diff
@@ -159,6 +159,166 @@ class Templates {
   </div>
     `;
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {string} link
+   * @param {srting} user
+   * @returns {string} HTML template
+   * @memberof Templates
+   */
+  static confirmEmail(link, user) {
+    const username = user || 'buddy';
```

Comment:
What is `buddy`?

Reply:
buddy is a friend, It was in the email template in case the user did not supply their first name during signup. First name is not required during signup. I am going to remove the buddy and use username instead since the username is required during signup. Thanks.

Reply:
I have made changes to PR. Thanks for the feedback.

Reply:
Username is no longer unique because of the fact that users could signup via social media, I would suggest you split the receivers email address thus (e.g - joy@yahoo.com = joy).
```suggestion
    const username = user || 'buddy'; 
    const username = user || email.split('@');
```